README.md

```diff
@@ -1,12 +1,11 @@
 ![logo](./tokbox-logo.png)
 
+# Accelerator Core iOS <br/>
+
 [![Build Status](https://travis-ci.org/opentok/accelerator-core-ios.svg?branch=master)](https://travis-ci.org/opentok/accelerator-core-ios)
-[![Version Status](https://img.shields.io/cocoapods/v/OTAcceleratorCore.svg)](https://cocoapods.org/pods/OTAcceleratorCore)
+[![GitHub release](https://img.shields.io/github/release/opentok/accelerator-core-ios.svg)](./README.md)
```

Comment:
Not sure why this is none. I would recommend to use the version from CocoaPods as it's automatic.